### ISSUE TITLE
[PW-5865] Python support for Local/Cloud-based Terminal API

### ISSUE THREAD

**Issue by paluigi** [Adyen/adyen-python-api-library#154]
**Is your feature request related to a problem? Please describe.**
For an integration we are working on, it's required to integrate our software with the Adyen Terminal API to process payments. For our embedded application, we use Python as core language and we would prefer to communicate with the terminals inside our app.

**Describe the solution you'd like**
Since Python is our language of choice, we'd be glad to see support for the Terminal API in this library (like .NET, Java, and Node)

**Describe alternatives you've considered**
We could integrate another layer in our app, and leverage the Node or .NET library. However, being on an embedded board with limited resources we would prefer not to do so.

**Additional context**
If integration of Terminal API is in the roadmpa, it'd be great to know...together with an approximate timeline.

**Comment by AlexandrosMor** [comment]
Hello @paluigi,

We are going to investigate to add the support for Local/Cloud-based Terminal API in python library. At the moment terminal api is supported by the other libraries by Adyen node, java and dotnet. 

kind regards,
Alexandros
Adyen

**Comment by allburov** [comment]
@jillingk I think @paluigi meant to have full support for new [Terminal API](https://docs.adyen.com/point-of-sale/terminal-api/terminal-api-reference), not just a few methods

I'm looking for it too, but don't see it in the library

**Comment by jillingk** [comment]
Hi @allburov,

You're totally right, I got confused by the PR that we linked to this issue. As of yet we only (partly) support the management of terminals for our backend in this library and not the direct terminal communication based on the Nexo standard. 

We're planning to make this library one to one with our own internal API specifications and to complement that we will also of course want to implement the terminal API, so hopefully we can provide this to you rather sooner than later. You're also always welcome to contribute yourself though!

Best, Jilling
Adyen
